content/components/recipes/details/arguments/BranchedAddonStudy.js

```diff
@@ -4,31 +4,42 @@ import { Tag } from "rsuite";
 
 import { tableFormatter } from "devtools/components/recipes/details/arguments/formatters";
 import GenericArguments from "devtools/components/recipes/details/arguments/GenericArguments";
+import { useExperimenterDetailsData } from "devtools/contexts/experimenterDetails";
 
 export default function BranchedAddonStudy({ data }) {
+  const experimenterData = useExperimenterDetailsData();
   return (
     <GenericArguments
       data={data.arguments}
       formatters={{
-        branches: tableFormatter(["slug", "ratio", "extensionApiId"], {
-          slug(index, value) {
-            return <code>{value}</code>;
-          },
-          ratio(index, value) {
-            return <code>{value}</code>;
-          },
-          extensionApiId(index, value) {
-            if (value) {
-              return <Tag color="violet">{value}</Tag>;
-            }
+        branches: tableFormatter(
+          ["slug", "description", "ratio", "extensionApiId"],
+          {
+            slug(index, value) {
+              return <code>{value}</code>;
+            },
+            description(index) {
+              const { slug } = data.arguments.branches[index];
+              return experimenterData.variants
+                ? experimenterData.variants[slug]
+                : null;
```

Comment:
This can done more concisely now with nullish coalescing and optional chaining:

```suggestion
              return experimenterData.variants?.[slug] ?? null;
```